_includes/manuals/3.14/1.2_release_notes.md

```diff
@@ -4,6 +4,17 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### IPV6 Support
+Satellite deployment and operation in IPv6 networks is now fully supported. Dual stack deployments have not been tested and are not supported.
+
+Provisioning over IPv6 is supported on bare metal hosts.
+For compute resources, you can define the machine outside Satellite on the compute resource and then you can provision the machine as bare metal in Satellite.
```

Comment:
Why would image mode not work on IPv6-only? If that isn't supported I wouldn't call IPv6-only fully supported.

Also, please fix the branding.